apps/system/components/public/mapa.tsx

```
@@ -0,0 +1,123 @@
+// components/SchoolMap.tsx
+"use client";
+
+import Link from "next/link";
+import { MapPin, LocateFixed, Map} from "lucide-react";
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)